[2025-10-16T10:03:01.242+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-16T10:03:01.258+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag2.dag2_bash_1 scheduled__2025-10-16T10:02:00+00:00 [queued]>
[2025-10-16T10:03:01.264+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag2.dag2_bash_1 scheduled__2025-10-16T10:02:00+00:00 [queued]>
[2025-10-16T10:03:01.265+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-16T10:03:01.277+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dag2_bash_1> on 2025-10-16 10:02:00+00:00
[2025-10-16T10:03:01.283+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=213) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-16T10:03:01.284+0000] {standard_task_runner.py:63} INFO - Started process 215 to run task
[2025-10-16T10:03:01.284+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag2', 'dag2_bash_1', 'scheduled__2025-10-16T10:02:00+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmpx4lw7r5w']
[2025-10-16T10:03:01.286+0000] {standard_task_runner.py:91} INFO - Job 50: Subtask dag2_bash_1
[2025-10-16T10:03:01.299+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-16T10:03:01.325+0000] {task_command.py:426} INFO - Running <TaskInstance: dag2.dag2_bash_1 scheduled__2025-10-16T10:02:00+00:00 [running]> on host bb861ccd12fb
[2025-10-16T10:03:01.400+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag2' AIRFLOW_CTX_TASK_ID='dag2_bash_1' AIRFLOW_CTX_EXECUTION_DATE='2025-10-16T10:02:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-16T10:02:00+00:00'
[2025-10-16T10:03:01.403+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-16T10:03:01.404+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-16T10:03:01.408+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "hello bash"']
[2025-10-16T10:03:01.419+0000] {subprocess.py:86} INFO - Output:
[2025-10-16T10:03:01.419+0000] {subprocess.py:93} INFO - hello bash
[2025-10-16T10:03:01.420+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-16T10:03:01.421+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-16T10:03:01.444+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag2, task_id=dag2_bash_1, run_id=scheduled__2025-10-16T10:02:00+00:00, execution_date=20251016T100200, start_date=20251016T100301, end_date=20251016T100301
[2025-10-16T10:03:01.500+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-16T10:03:01.515+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-16T10:03:01.518+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
